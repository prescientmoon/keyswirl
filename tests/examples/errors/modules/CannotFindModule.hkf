module exporting () 

import foo.bar.goo as FBG -- oops, no such module
