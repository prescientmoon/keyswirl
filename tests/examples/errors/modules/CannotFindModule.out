[error ModuleNotFound]: Module "foo.bar.goo" not found
     +--> errors.modules.CannotFindModule@3:8-3:19
     |
   3 | import foo.bar.goo as FBG -- oops, no such module
     :        ^---------- 
     :        `- This module does not exist
-----+
